Графика для всех страниц

eggsandbaky opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1808931/tasks/5)

eggsandbaky committed:
Добавил изображения

eggsandbaky committed:
Исправил имена

Исправил имена файлов и директорий в соответствии с кодгайдом

eggsandbaky committed:
Исправил пути

maxzbest commented on catalog.html:
```diff
@@ -137,7 +137,7 @@ <h3 class="visually-hidden">Параметры вывода результато
           <ul class="result-list">
             <li class="result-item">
-              <img class="result-image" src="" alt="Гостиница Amara Resort & Spa">
+              <img class="result-image" src="images/catalog/items/amara-resort-&-spa.jpg" width="299" height="206" alt="Гостиница Amara Resort & Spa">
```
В макете по 300 у них ширина

eggsandbaky replied:
Видимо у меня был старый макет. Исправил.

maxzbest commented on index.html:
```diff
@@ -135,7 +135,7 @@ <h2 class="visually-hidden">Наши контакты</h2>
+          <img class="html-academy-logo" src="images/htmlacademy.svg" width="115" height="34" alt="логотип HTML Academy">
         </a>
       </section>
     </footer>
```
+ моменты описанные мной в чатике

eggsandbaky committed:
Изменил титл страницы каталога

Поменял титл внутренней страницы на более информативный

eggsandbaky committed:
Исправил размер превьюшек

Привел размер изображений в соответствии с новым макетом.

eggsandbaky committed:
Добавил фавикон

Добавил фавикон на страницы с помощью сервиса "https://realfavicongenerator.net/"

eggsandbaky committed:
Оптимизировал svg

Оптимизировал все svg изображения с помощью svgo

eggsandbaky committed:
Оптимизировал изображения
